### PR TITLE
updating mixins and color sections in sass file organization

### PR DESCRIPTION
**Pull request by iruzevic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
 # frontend-handbook
+
+#How to update
+* Create a new branch from `maste`.
```
Comment (Stram) [inline README.md]: master*

File: SASS Styleguide/File organization.md [hunk 2]
```diff
@@ -255,7 +259,7 @@ Example usage of a global placeholder.
 
 ### Mixins
 
-The mixins file should contain any global mixin that can help organize sass better.
+The mixins file should contain any global mixin that can help organize sass better. If your project contains a lot of mixins it is reccomended that you put them in a separate files and groupe according to their purpose. Then put all those filed inside a new folder called **mixins**.
```
Comment (Stram) [inline SASS Styleguide/File organization.md]: recommended* group* filed*

File: README.md (hunk 1 above)
Comment (dingo-d) [inline README.md]: Typo `master` :smile:

File: SASS Styleguide/File organization.md [hunk 3]
```diff
@@ -316,6 +332,7 @@ e.g.
 }
 
 ```
+If you project uses postCss there are libs that will make all this font-face declarations for you.
```
Comment (Stram) [inline SASS Styleguide/File organization.md]: PostCSS*

File: README.md (hunk 1 above)
Comment (iruzevic) [reply README.md]: 👍

Comment (iruzevic) [reply README.md]: 👍

File: SASS Styleguide/File organization.md (hunk 2 above)
Comment (iruzevic) [reply SASS Styleguide/File organization.md]: 👍

File: SASS Styleguide/File organization.md (hunk 3 above)
Comment (iruzevic) [reply SASS Styleguide/File organization.md]: 👍

File: SASS Styleguide/File organization.md [hunk 4]
```diff
@@ -128,12 +128,16 @@ When you're working on a project with `_shame.scss` always check the file when y
 The colors file should only contain color variables of the base and global kind.
 All colors for blocks or elements should go into their local file on the top.
 
-Base colors are actual color values used on the site, they will be used by both global colors and block-elements colors. Syntax and example:
+Base colors are actual color values used on the site, they will be used by both global colors and block-elements colors. 
+
+When naming color variables avoid using names that correspond to the actual colors, better use some thing like `silver`, `wood`, `mercury`, etc...
```
Comment (tomislav-s) [inline SASS Styleguide/File organization.md]: Silver is actually a color: https://en.wikipedia.org/wiki/Silver_(color)

So it's best if we remove it or have a different example here

Comment (iruzevic) [reply SASS Styleguide/File organization.md]: 👍

File: SASS Styleguide/File organization.md [hunk 5]
```diff
@@ -316,6 +332,7 @@ e.g.
 }
 
 ```
+If you project uses PostCSS there are libs that will make all these font-face declarations for you.
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply SASS Styleguide/File organization.md]: 👍

File: SASS Styleguide/File organization.md [hunk 6]
```diff
@@ -255,7 +259,7 @@ Example usage of a global placeholder.
 
 ### Mixins
 
-The mixins file should contain any global mixin that can help organize sass better.
+The mixins file should contain any global mixin that can help organize sass better. If your project contains a lot of mixins it is recommended that you put them in a separate files and group according to their purpose. Then put all those files inside a new folder called **mixins**.
```
[the remark this answers is not in the document]
Comment (iruzevic) [reply SASS Styleguide/File organization.md]: 👍

**Comment by iruzevic** [comment]
all corrections are addressed, please check.

**Pull request merged**